AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/allocate-hearings/allocate-hearings.component.html

```diff
@@ -149,6 +149,9 @@
           >
             <div class="more-than-three">3+</div>
           </span>
+          <span appTooltip [text]="'Hearing clashes with user non-availability'" [colour]="'blue'" colour="grey">
```

Comment:
If you see on line 107 you'll understand what I mean by another column.

Each icon should be in it's own column like so
```<th scope="col" class="govuk-table__header no-border"></th>```

and
```  <td class="govuk-table__cell no-border"> <icon> </td>```